pkg/agent/inject.go

```diff
@@ -196,7 +196,10 @@ func downloadAgentLocally(tryDownloadURL, targetArch string, log log.Logger) (st
 		return agentPath, nil
 	}
 
-	resp, err := devpodhttp.GetHTTPClient().Get(tryDownloadURL + "/devpod-linux-" + targetArch)
+	fullDownloadURL := tryDownloadURL + "/devpod-linux-" + targetArch
+	log.Infof("Attempting to download DevPod agent from: %s", fullDownloadURL)
```

Comment:
What do you think about making this a debug log?
Imo it's not something everyone needs all of the time when creating workspace but rather when troubleshooting potential issues

Reply:
Sounds good, I just pushed another commit.